api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -97,6 +112,7 @@
   private static final String LOGGED_IN_USER_EMAIL = "bob@gmail.com";
 
   @TestConfiguration
+
```

Comment:
Please revert, `@TestConfiguration` annotates the below class, so the newline is misleading